bareos/bconsole.sls

```diff
@@ -31,5 +32,5 @@ bareos_bconsole:
     - user: root
     - group: root
     - require:
-      - pkg: bareos_bconsole
+      - pkg: bareos-bconsole
```

Comment:
You changed here the dependency on the *resource* to a dependency on the *real package*. This way, if the package name changes for whatever reason, the dependency will fail. Perhaps it's better if you keep the depedency on the resource, like

```yaml
      - pkg: install_bconsole_package
```

so we depend on the resource success instead that depending on the existance of certain package? This abstracts the formula for the name the distros give to packages (ie, bconsole, bareos-console, bareos-bconsole, etc.)